common_gaku_dependencies.rb

```diff
@@ -1,12 +1,10 @@
-source 'http://production.s3.rubygems.org'
+source 'http://rubygems.org'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.